dsc/src/subcommand.rs

```diff
@@ -359,16 +358,20 @@ pub fn resource(subcommand: &ResourceSubCommand, stdin: &Option<String>) {
                                 }
                             }
                         }
-                        if !found {
-                            continue;
-                        }
+                        if !found { continue; }
                     }
+
+                    methods = "get".to_string();
```

Comment:
Should probably still keep a check here for `get` as the discussion of whether it must be mandatory might change in the future.

Reply:
That makes sense - what do you suggest checking? 
The get field of the struct isn't optional so the `is_some()` check doesn't apply.